README.md

```diff
@@ -127,9 +127,12 @@ Sometimes, an app wants to specify that the js/css files are in an absolute path
 Since Pulito just returns a Webpack object, the output of `commonBuilder` can be
 modified directly, like:
 
-    const { commonBuilder } = require('pulito');
-    module.exports = commonBuilder(__dirname);
-    module.exports.output.publicPath='/static/';
+    const commonBuilder = require('pulito');
+    module.exports = (env, argv) => {
```

Comment:
I'm doubly confused - module.exports is an object OR a function that returns an object OR?  Is that new to webpack 4?

Reply:
Yes, it can be either obj or function. Not sure if that's new in webpack 4.